File: Python/For e While/4.py
#Prova 2 do JUDE
#Questão 2

Entrada = input()
Valores = Entrada.split()
N = int(Valores[0])
Q = int(Valores[1])
Valor=0

for x in range(N):
    Valor = Valor + Q
    Q=Q*2
print (Valor)

File: Cpp/For e While/4.cpp
//Lista 2 do JUDE
//Questão 4

#include <iostream>
using namespace std;

int main() 
{
int N, Valor;
cin >> N;
Valor = N;

while (Valor % 2 == 0)
{
Valor = Valor/2;
}

if (Valor == 1)
{
    cout << "Dattebayo" << endl;
}
else
{
    cout << "Tururuuu" << endl;
}
return 0;
}